app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find_by(id: params[:id])
+    if @user.nil?
+      head :not_found
+    end
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
```

Comment:
How are `new` and `create` here different than in the `SessionsController`? Do you need both for this site?